app/models/concerns/orchestration/compute.rb

```diff
@@ -154,7 +154,12 @@ def delCompute
 
   def setComputePowerUp
     logger.info "Powering up Compute instance for #{name}"
-    compute_resource.start_vm uuid
+    if compute_resource.provider == 'Ovirt'
```

Comment:
As mentioned above, this is supposed to be a generic method for all compute resources so Ovirt logic cannot really fit in here, `start_vm` should handle this instead.